content/docs/glossary.md

```diff
@@ -144,8 +143,7 @@ information on where each chart can be downloaded from. (Many chart
 repositories serve the charts as well as the `index.yaml` file.)
 
 A Helm client can point to zero or more chart repositories. By default,
-Helm clients point to the `stable` official Kubernetes chart
-repository.
+Helm clients are not configured with any chart repositories.
```

Comment:
Can we add an additional sentence, stating

> Chart repositories can be added at any time using the `helm repo add` command.

Reply:
Definitely a good idea, adding now